rllib/contrib/maddpg/maddpg.py

```diff
@@ -171,10 +171,28 @@ def collect_metrics(trainer):
     return result
 
 
+def add_maddpg_postprocessing(config):
+    """Add the before learn on batch hook.
+
+    This hook is called explicitly prior to TrainOneStep() in the execution
+    setups for DQN and APEX.
```

Comment:
It's a bit unfortunate we have to add this hook, but the alternative to define a new execution plan just for MADDPG is probably too messy.

Reply:
Yeah, probably not necessary right now.